requirements.txt

```diff
@@ -1,23 +1,23 @@
--e .
-aws-cdk.aws-cloudformation~=1.109.0
-aws-cdk.aws-cloudwatch~=1.109.0
-aws-cdk.aws-codebuild~=1.109.0
-aws-cdk.aws-codecommit~=1.109.0
-aws-cdk.aws-codedeploy~=1.109.0
-aws-cdk.aws-codeguruprofiler~=1.109.0
-aws-cdk.aws-codepipeline~=1.109.0
-aws-cdk.aws-codepipeline-actions~=1.109.0
-aws-cdk.aws-dynamodb~=1.109.0
-aws-cdk.aws-ec2~=1.109.0
-aws-cdk.aws-iam~=1.109.0
-aws-cdk.aws-kms~=1.109.0
-aws-cdk.aws-logs~=1.109.0
-aws-cdk.aws-s3~=1.109.0
-aws-cdk.aws-s3-assets~=1.109.0
-aws-cdk.aws-s3-deployment~=1.109.0
-aws-cdk.aws-s3-notifications~=1.109.0
-aws-cdk.aws-secretsmanager~=1.109.0
-aws-cdk.aws-ssm~=1.109.0
-aws-cdk.core~=1.109.0
-aws-cdk.pipelines~=1.109.0
+aws-cdk.aws-cloudformation~=1.181.1
```

Comment:
@bwighane I am just going to merge and see but I suspect the adjustment of these versions might break something.

Reply:
I understand. 'cdk synth' worked after making corrections to the requirements. Let me know how that goes.

Reply:
sure!